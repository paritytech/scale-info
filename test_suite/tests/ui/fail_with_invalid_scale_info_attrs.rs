use info::{self as scale_info};
use scale_info::TypeInfo;
use scale::Encode;

#[derive(TypeInfo, Encode)]
#[scale_info(foo)]
struct InvalidKeywordInScaleInfoAttr {
    a: u8,
    b: u16,
}

fn main() {}
