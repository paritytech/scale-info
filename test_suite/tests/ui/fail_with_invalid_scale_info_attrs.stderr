error: expected one of: `bounds`, `skip_type_params`, `capture_docs`, `crate`, `replace_segment`
 --> tests/ui/fail_with_invalid_scale_info_attrs.rs:6:14
  |
6 | #[scale_info(foo)]
  |              ^^^
